airbyte-integrations/connectors/source-bing-ads/source_bing_ads/source.py

```diff
@@ -597,16 +597,16 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             Campaigns(client, config),
         ]
 
-        if config["hourly_reports"] or config["daily_reports"] or config["weekly_reports"] or config["monthly_reports"]:
-            streams.append(BudgetSummaryReport(client, config))
-
-        if config["hourly_reports"]:
-            streams.extend([c(client, config) for c in self.get_report_streams("Hourly")])
-        if config["daily_reports"]:
-            streams.extend([c(client, config) for c in self.get_report_streams("Daily")])
-        if config["weekly_reports"]:
-            streams.extend([c(client, config) for c in self.get_report_streams("Weekly")])
-        if config["monthly_reports"]:
-            streams.extend([c(client, config) for c in self.get_report_streams("Monthly")])
+        config['hourly_reports'] = True
```

Comment:
we can remove this since nothing is using it now

Reply:
it is done in latest commit

Reply:
I am looking on https://github.com/airbytehq/airbyte/pull/13801/files and I still see us setting `config['hourly_reports'] = True` -- did you push the latest local commit?

Reply:
You absolutely right!  I was fixed